ci-operator/step-registry/optional-operators/cvp-common/apply-secrets/optional-operators-cvp-common-apply-secrets-commands.sh

```diff
@@ -94,9 +94,16 @@ LIST_OF_KUBEOBJECTS=$(find "${CUSTOM_KUBEOBJECTS_PATH}"* -type f -name kube_obje
 if [ -n "${LIST_OF_KUBEOBJECTS[0]}" ] ; then
     for i in "${LIST_OF_KUBEOBJECTS[@]}"
     do
-        echo "The following custom kubeobject has been found ! $i"
-        echo "Applying kube_objects on to the Namespace $OO_INSTALL_NAMESPACE"
-        oc apply -f "$i" -n "$OO_INSTALL_NAMESPACE"
+	echo "The following custom kubeobject has been found ! $i"
+	echo "Check if the namespace is mentioned inside the kubeobjects"
+        if grep -inq "namespace" $i
+        then
+            echo "Namespace found in kubeobjects. Applying directly"
+	    oc apply -f "$i"
```

Comment:
What if namespace was mentioned in comment or in some text field?
Should we really skip `-n "$OO_INSTALL_NAMESPACE"` in such case?

Reply:
What @lslebodn means is, the case-insensitive string `namespace` could appear in a comment or a text field and so not necessarily indicate there is a `Namespace` object contained within the yaml document. 

I suggest we grep for a pattern like `kind:\s+Namespace`  or something like that so we can be more certain there is a `Namespace` object defined there.